app/src/components/main/attributeList/editableAttribute/editableAttribute.jsx

```diff
@@ -0,0 +1,44 @@
+import { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Attribute } from './attribute';
+import { AttributeEditor } from './attributeEditor';
+
+export class EditableAttribute extends Component {
+  static propTypes = {
+    attribute: PropTypes.object,
+    editMode: PropTypes.bool,
+    onEdit: PropTypes.func,
+    onRemove: PropTypes.func,
+    onChange: PropTypes.func,
+  };
+
+  static defaultProps = {
+    attribute: {},
+    editMode: false,
+    onEdit: () => {},
+    onRemove: () => {},
+    onChange: () => {},
+  };
+
+  enterEditMode = () => this.props.onEdit(this.props.attribute);
+
+  exitEditMode = () => this.props.onEdit(null);
+
+  calculateFormName = (attribute) =>
+    attribute ? `attributesEditor__${attribute.key}_${attribute.value}` : 'attributesEditor';
```

Comment:
`attribute` is `{}` by default, so you will get `attributesEditor__undefined_undefined`

Reply:
fixed